src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs

```diff
@@ -1275,5 +1309,30 @@ private static void UpdateValidationDetailsForAddress(Page page, List<Validation
             }
 
         }
+
+        public async Task<FinancialExpiredViewModel> IsFinancialExpired(string epaoId, string redirAction, string redirController)
+        {
+            var model = new FinancialExpiredViewModel();
+            var organisation = await _orgApiClient.GetEpaOrganisation(epaoId);
+            if (organisation != null)
+            {
+                var orgDataFinancialExempt = organisation.OrganisationData?.FHADetails?.FinancialExempt;
```

Comment:
I think this here can be improved, the `GetEpaOrganisation` call returns the `organisation.FinancialReviewStatus` of Exempt or required which is done Api side with the same tests.

Can this not be:

```
                    if (organisation.FinancialReviewStatus != ApplyTypes.FinancialReviewStatus.Exempt)
                    {
                        model.FinancialInfoStage1Expired = true;
                        model.FinancialAssessmentUrl = this.Url.Action(redirAction, redirController);
                    }
```
?

Also this method doesn't need to return a financial assessment url , both the two instances that call this method only care about `FinancialInfoStage1Expired` being true or false, so the method could be reduced to only return this value. And when this is taken into account, the actual helper method could be removed entirely and where it's called be replaced with
```
var organisation = await _orgApiClient.GetEpaOrganisation(epaoId);
var financialExpired = organisation?.FinancialReviewStatus == ApplyTypes.FinancialReviewStatus.Exempt ?? false;
```

As in the areas that use the boolean, only if it's true does the `FinancialExpiredModel` come into play. so for example:
```
            var financialExpired = await IsFinancialExpired(epaoid, "StartOrResumeApplication", "Application");

            ApplicationResponseViewModel applicationsResponseVm = new ApplicationResponseViewModel();
            applicationsResponseVm.ApplicationResponse = existingApplications;
            applicationsResponseVm.FinancialInfoStage1Expired = financialExpired.FinancialInfoStage1Expired;
            applicationsResponseVm.FinancialAssessmentUrl = financialExpired.FinancialAssessmentUrl;
```
Becomes
```
            var organisation = await _orgApiClient.GetEpaOrganisation(epaoId);
            var financialExpired = organisation?.FinancialReviewStatus == ApplyTypes.FinancialReviewStatus.Exempt ?? false;            
            ApplicationResponseViewModel applicationsResponseVm = new ApplicationResponseViewModel();
            applicationsResponseVm.ApplicationResponse = existingApplications;
            applicationsResponseVm.FinancialInfoStage1Expired = financialExpired;
            applicationsResponseVm.FinancialAssessmentUrl = this.Url.Action("StartOrResumeApplication", "Application")
```

Reply:
I've made changes for Uma to Test as @anilkdhiman6  is away and the release was scheduled for Monday. Can talk through when you're back Anil.